CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # node-google-dfp-wrapper
 
+### 1.3.0
+- Adds `getCriteriaValues` for querying criteria values without extracting id.
+- Adds `getCriteriaValueId` for querying criteria values and extracting id.
```

Comment:
Get `getCriteriaValueId` returns the id of a single value. `getCriteriaValue` does the same, but should be fixed in a major release at some point. `getCriteriaValues` returns many values as full objects, not just their id